perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -15,24 +15,35 @@ cd "${KOKORO_ARTIFACTS_DIR}/github/gcsfuse"
 # Get the latest commitId of yesterday in the log file
 commitId=$(git log --before='yesterday 23:59:59' --max-count=1 --pretty=%H)
 git checkout $commitId
+
+# Building and installing gcsfuse
```

Comment:
Can we use echo statement here as well ?

Reply:
fixed